Mathlib/Analysis/NormedSpace/Star/GelfandDuality.lean

```diff
@@ -139,6 +139,8 @@ theorem gelfandTransform_map_star (a : A) :
 
 variable (A)
 
+attribute [-instance] instIsNoetherian_of_isNoetherianRing_of_finite
```

Comment:
Was this previously not an instance because it formed a typeclass loop? Is this evidence that typeclass loops are still problematic even in Lean 4?

Reply:
Oh, our comments crossed. I think the answer is "yes".